### PR TITLE
DX-2021 Voice, Answering Machine Detection

### PR DESCRIPTION
**Pull request by ajrice6713**

### CODE REVIEW

File: src/main/java/com/bandwidth/messaging/controllers/APIController.java [hunk 1]
```diff
@@ -157,7 +157,8 @@ private HttpRequest buildListMediaRequest(
             throw new MessagingException("400 Request is malformed or invalid", context);
         }
         if (responseCode == 401) {
-            throw new MessagingException("401 The specified user does not have access to the account", context);
+            throw new MessagingException(
```
Comment (ckoegel) [inline src/main/java/com/bandwidth/messaging/controllers/APIController.java:160]: is there a reason all of these have been moved to a newline? My Java experience is super limited so I might be missing something but it seems pointless.

File: src/main/java/com/bandwidth/messaging/controllers/APIController.java [hunk 2]
```diff
@@ -157,7 +157,8 @@ private HttpRequest buildListMediaRequest(
             throw new MessagingException("400 Request is malformed or invalid", context);
         }
         if (responseCode == 401) {
-            throw new MessagingException("401 The specified user does not have access to the account", context);
+            throw new MessagingException(
+                    "401 The specified user does not have access to the account", context);
```
Comment (ckoegel) [inline src/main/java/com/bandwidth/messaging/controllers/APIController.java:161]: is there a reason all of these have been moved to a newline? My Java experience is super limited so I might be missing something but it seems pointless.

Comment (ajrice6713) [reply src/main/java/com/bandwidth/messaging/controllers/APIController.java:161]: #justApiMaticThings 

I didnt write any of that its all autogenerated

Comment (ckoegel) [reply src/main/java/com/bandwidth/messaging/controllers/APIController.java:161]: poggers, we love apimatic

**Review by ckoegel — APPROVED** [review]

**Review by hamermike — APPROVED** [review]

**Pull request merged**